apps/confessions-client/components/Confessions.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { listConfessions } from "../src/api";
+  
+export function Confessions({
+    accessToken,
+    newConfession,
+}: {
+    accessToken: string | null;
+    newConfession: string | undefined;
+}) {
+  const [confessions, setConfessions] = useState<any>(null);
+
+  useEffect(() => {
+    if (!accessToken) {
+      setConfessions(null)
+      return;
+    }
+  
+    (async () => {
+      // TODO: paging
```

Comment:
we should implement this before we launch, so that we can fire and forget

Reply:
won't get to this until later this week or early next week